pipeline/full_pipeline.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2021, Blue Brain Project, EPFL
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script that run the full pipeline."""
+from __future__ import annotations
+
+import argparse
+import logging
+import sys
+from pathlib import Path
+
+from atldld.utils import get_experiment_list_from_gene
+
+logger = logging.getLogger("full-pipeline")
```

Comment:
I don't think we ever use it. It would be nice to have a couple of messages across the script.

Reply:
Do you mean that we do not use the `logger` ?